distributed/distributed.yaml

```
@@ -198,6 +198,11 @@ distributed:
 
   diagnostics:
     nvml: True
+    computations:
+      max-history: 100
+      ignore-modules:
+        - distributed
+        - dask
```

```suggestion
        - dask
        - xarray
        - cudf
        - cuml
        - prefect
        - xgboost
```

Let's add a few.  I'm curious, will this work with anything that partially matches, for example with `xgboost.dask` or `dask-xgboost` already be ignored because they include the term "dask"?

---

`xgboost.dask` will be ignored already. I'm using a simple regex, i.e. if `dask` is found in the module name, we continue our walk. If this is too loose, we'll need to match for explicit module names with less magic